lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/Parameters/AzureBlobStorageAccountKey.cs

```diff
@@ -19,10 +19,10 @@
 
 namespace Org.Apache.REEF.IO.FileSystem.AzureBlob.Parameters
 {
-    [NamedParameter("The connection string for Azure Storage")]
-    public sealed class AzureStorageConnectionString : Name<string>
+    [NamedParameter("The Azure Blob Storage account key")]
+    public sealed class AzureBlobStorageAccountKey : Name<string>
```

Comment:
>AzureBlobStorageAccountKey [](start = 24, length = 26)

FYI - This is an API surface breaking change. I much prefer this than the connection string. However, the REEF team may want us to still support the legacy ConnectionString parameter so be prepared to support both if necessary.